doc/changes/changes_2.2.0.md

```diff
@@ -0,0 +1,45 @@
+# Virtual Schema for BigQuery 2.2.0, released 2023-03-16
+
+Code name: Dependency Upgrade on top of 2.1.1
+
+## Summary
+
+Updated dependencies to fix vulnerability CVE-2022-45688 in test dependency [org.json:json:jar:20220924](https://ossindex.sonatype.org/component/pkg:maven/org.json/json@20220924?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1).
```

Comment:
Why do you increment the minor version for security updates? The patch version should be enough.